scripts/build/build/targets.py

```diff
@@ -477,6 +473,8 @@ def BuildTizenTarget():
         TargetPart('all-clusters-minimal', app=TizenApp.ALL_CLUSTERS_MINIMAL),
         TargetPart('chip-tool', app=TizenApp.CHIP_TOOL),
         TargetPart('light', app=TizenApp.LIGHT),
+        # test driver app
+        TargetPart('tests', app=TizenApp.TESTS).OnlyIfRe("-qemu"),
```

Comment:
It seems we also want a '-no-ble` conditional here. There is no good support for multiple conditions here, however qemu is the board, so no-ble would always follow it, so I assume `OnlyIfRe("-qemu-.*-no-ble")` seems like a correct condition.

Reply:
The idea here was to allow the `tests` target to be run on QEMU "board" only, but with all other possible variants. So currently we have "tizen-qemu-tests-no-ble" in CI, because current QEMU setup does not support BLE. But also, we can tests things like "tizen-qemu-tests-no-ble-no-wifi", or simply "tizen-qemu-tests". However, in regards to your comment about 'qemu' vs 'arm' and what is the difference, I will drop the concept of QEMU board and simply go with 'arm' as an architecture-only constraint.